src/Common/AzurePSCmdlet.cs

```diff
@@ -551,6 +551,19 @@ protected new void WriteWarning(string text)
             base.WriteWarning(text);
         }
 
+        protected new void WriteInformation(object messageData, string[] tags)
+        {
+            FlushDebugMessages();
+            base.WriteInformation(messageData, tags);
+        }
+
+        protected void WriteInformation(string text)
+        {
+            FlushDebugMessages();
+            HostInformationMessage message = new HostInformationMessage { Message = ansiCodePrefix + text + ansiCodeSuffix, NoNewLine = false };
```

Comment:
Using the ansiCodePrefix and suffix will result in a specific format. Are you sure we want all _informational_ messages to be in this format?

Reply:
If no ansicodePrefix and suffix, the format of informational message and output is same. I think we need to distinguish them explicitly. 

I thought WriteInformation(string text) is a predefined shortcut function for another one, if developer needs another format, they can use   WriteInformation(object, string[])